sql/scan.trace

```diff
@@ -0,0 +1,12 @@
+statement ok
+CREATE TABLE t (k INT PRIMARY KEY)
+
+statement ok
+INSERT INTO t VALUES (1)
+
+traceon trace.out
```

Comment:
Did you mean to check this in? (maybe we should add a grep in `make check` for any test file with `traceon`)